CMakeLists.txt

```
@@ -74,6 +74,13 @@ if(HDF5_IS_PARALLEL)
   message(STATUS "Using parallel HDF5")
 endif()
 
+# Version 1.12 of HDF5 deprecates the H5Oget_info_by_idx() interface.
+# Thus, we give these flags to allow usage of the old interface in newer
+# versions of HDF5.
+if(${HDF5_VERSION} VERSION_GREATER_EQUAL "1.12.0") 
```

`VERSION_GREATER_EQUAL` didn't make it in to CMake until version 3.7. Right now our CMakeLists.txt file requires 3.3 or higher. The following should be equivalent 
```suggestion
if(NOT (${HDF5_VERSION} VERSION_LESS 1.12.0))
```

---

Ah, good eyes!